README.md

```diff
@@ -58,6 +58,20 @@ Not to be confused with [leveljs](https://www.npmjs.com/package/leveljs).
 
 This library is best used with [browserify](http://browserify.org).
 
+## API
+
+### `db = leveljs(location[, options])`
+Returns a new `leveljs` instance. `location` is a String pointing to the IndexedDB location to be opened.
```

Comment:
```diff
-`location` is a String pointing to the IndexedDB location to be opened
+`location` is the string name of the IndexedDB object store to be opened
```

Reply:
Nah, that's not quite it either. It also controls the IndexedDB database name.

Reply:
@vweevers Think about it for a bit. I'll do other stuff meanwhile.

Reply:
How about: `location` is the string name of the [`IDBDatabase`](https://developer.mozilla.org/en-US/docs/Web/API/IDBDatabase) to be opened, as well as the object store within that database. The database name will be prefixed with `options.prefix`.